test_sets/policies_smoke.txt

```diff
@@ -0,0 +1,6 @@
+./test_scripts/Polices/build_options/001_ATF_P_TC_SDL_Build_Flag_DEXTENDED_POLICY_PROPRIETARY.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_kilometers.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_days.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_IGN_cycles.lua
+./test_scripts/Polices/build_options/ATF_P_TC_PTU_Transfer_Several_Apps_Different_HMI_Levels_PROPRIETARY.lua
+./test_scripts/Polices/build_options/ATF_P_PTS_creation_rule_PROPRIETARY.lua
```

Comment:
@dboltovskyi, the checks for preloaded_pt.json are performed when SDL starts even before HMI initialization and are not affected by policy flow.
Please check again.